plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/hand-picked-products-control.tsx

```diff
@@ -158,31 +186,20 @@ const HandPickedProductsControl = ( {
 
 	return (
 		<ToolsPanelItem
-			label={ __( 'Hand-picked Products', 'woocommerce' ) }
+			label={ __( 'Hand-Picked Products', 'woocommerce' ) }
 			hasValue={ () => !! selectedProductIds?.length }
 			onDeselect={ deselectCallback }
 			resetAllFilter={ deselectCallback }
 		>
-			<FormTokenField
-				displayTransform={ transformTokenIntoProductName }
-				label={ __( 'Hand-picked Products', 'woocommerce' ) }
-				onChange={ onTokenChange }
-				onInputChange={ isLargeCatalog ? handleSearch : undefined }
-				suggestions={ suggestions }
-				// @ts-expect-error Using experimental features
-				__experimentalValidateInput={ ( value: string ) =>
-					productsMap.has( value )
-				}
-				value={
-					! productsMap.size
-						? [ __( 'Loading…', 'woocommerce' ) ]
-						: selectedProductIds || []
-				}
-				__experimentalExpandOnFocus={ true }
-				__experimentalShowHowTo={ false }
+			<HandPickedProductsControlField
+				query={ query }
+				trackInteraction={ trackInteraction }
+				setQueryAttribute={ setQueryAttribute }
 			/>
 		</ToolsPanelItem>
 	);
 };
 
+HandPickedProductsControl.collectionSpecific = true;
+
```

Comment:
**Question:** What's the purpose of `collectionSpecific`? I can't find any usage of it in the codebase 🤔

Reply:
CC: @WunderBart

Reply:
Ah, nice catch - that's just a leftover from something I tested earlier. Will remove.

Reply:
Removed in https://github.com/woocommerce/woocommerce/pull/51141/commits/f8c4a106eeb0b7a0991e88cba77b1f6e9ba15e30.